sos/report/plugins/hbn.py

```diff
@@ -12,13 +12,30 @@
 
 class HBN(Plugin, IndependentPlugin):
 
-    short_desc = 'HBN (Host Based Network)'
+    short_desc = 'HBN (Host Based Networking)'
     plugin_name = "hbn"
+    vsctl = "ovs-vsctl"
     packages = ('hbn-repo',)
 
     def setup(self):
         self.add_copy_spec([
             '/etc/mellanox',
             '/var/log/doca/hbn',
             '/var/lib/hbn/etc/cumulus',
+            '/var/log/containers',
+            self.path_join('/var/log', 'sfc-install.log'),
+            self.path_join('/tmp', 'sf_devices'),
+            self.path_join('/tmp', 'sfr_devices'),
+            self.path_join('/tmp', 'sfc-activated'),
+        ])
+
+        self.add_journal(units="sfc")
+        self.add_journal(units="sfc-state-propagation")
+
+        self.add_cmd_output([
+            f"{self.vsctl} -t 5 list Open_vSwitch .",
```

Comment:
isn't that collected as part of the openvswitch plugin? if not then it should be added there. 
The recommendation is not to duplicate information between plugins. 
users usually not run only a specific plugin.

Reply:
Ok. 
I assumed we can ask customers to run just one plugin for hbn troubleshooting and it can be have enough info.
Also, I noticed that openvswitch does not run this specific command, but a bunch of other commands which provides a subset of the info.

Reply:
it should not happen, and as i said the recommendation is not to collect the same informations from multiple plugins.

Reply:
https://github.com/sosreport/sos/wiki/How-to-Write-a-Plugin#what-are-sos-plugins